internal/directives/schemas/json-update-config.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "JSONUpdateConfig",
+
+  "definitions": {
+
+    "jsonUpdate": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "The key whose value needs to be updated. For nested values, use a JSON dot notation path.",
+          "minLength": 1
+        },
+        "value": {
+          "type": ["string", "number", "boolean", "object", "array", "null"],
+          "description": "The new value for the specified key."
+        }
```

Comment:
Because the codegen used to create Go structs from these schemas is a bit temperamental, I suggest creating a definition for a type `jsonValue` and using that here. It will improve the generated code so that the struct is called `JsonValue`, which looks less generic than `Value`. It will avoid the need to refactor schemas and/or step implementations in the future if/when the need for some _other_ kind of "value" type arises.

Reply:
Did a bit of experimentation for you. If you omit the `type` entirely, you'll end up with this:

```
type JSONUpdate struct {
	// The key whose value needs to be updated. For nested values, use a JSON dot notation path.
	Key string `json:"key"`
	// The new value for the specified key.
	Value interface{} `json:"value"`
}
```

And that should be quite a bit more pleasant to work with and should help resolve many of my other comments.